### ISSUE TITLE
No extra Symbol for Ignore Trainer Input

### ISSUE THREAD

**Issue by hrenz** [FrSkyRC/ETHOS-Feedback-Community#4795]
ther is no extra Symbol for this Option

![image](https://github.com/user-attachments/assets/08f0c36e-af0f-4799-9334-16f05d5f8b31)

All other Options have extra Symbols

---------
Aand I think it should be Trainee Input and not Trainer Input

**Comment by bsongis-frsky** [comment]
Agree for Trainee input. What do you propose as an extra symbol?

**Comment by hrenz** [comment]
symbol examples for Ignore trainee input  Nr. 237
![image](https://github.com/user-attachments/assets/92d138c3-db05-4af1-abd7-67924c171419)

**Comment by lthole** [comment]
I suggest to use the double right arrow with stroke:

![Image](https://github.com/user-attachments/assets/ef8a1669-33da-4d1b-a7ee-74022ab45e51)

**Comment by bsongis-frsky** [comment]
Proposal 1 :

<img width="800" height="480" alt="Image" src="https://github.com/user-attachments/assets/bc7dc6a4-1ab6-4634-96ac-1c53797f4bc2" />

In case the Input also has the negative option:

<img width="800" height="480" alt="Image" src="https://github.com/user-attachments/assets/469d12df-6a16-40b7-bd97-2a73f07065ee" />

**Comment by lthole** [comment]
Makes sense!

**Comment by bsongis** [comment]
The U+21CF is not available in the Ethos font :(

**Comment by bsongis** [comment]
Proposal 2:

<img width="800" height="480" alt="Image" src="https://github.com/user-attachments/assets/c42d24ef-1a85-4b68-a4d8-059a1f0a5770" />

In case the Input also has the negative option:

<img width="800" height="480" alt="Image" src="https://github.com/user-attachments/assets/30b439ba-6a97-4a4b-b33d-a29d45886c5b" />

**Comment by lthole** [comment]
The advantage of this one is that it won’t be mistaken for a typo, which ‘x’ could be.

**Comment by bsongis-frsky** [comment]
True. I will choose this one too for now. If anybody has a better idea, feel free to propose!
